java/modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/utils/sslcert/ocsp/OCSPVerifier.java

```diff
@@ -217,7 +219,9 @@ private OCSPReq generateOCSPRequest(X509Certificate issuerCert, BigInteger seria
      * @throws CertificateVerificationException
      *
      */
-    private List<String> getAIALocations(X509Certificate cert) throws CertificateVerificationException {
+
+    @SuppressWarnings({"startswith"}) //FALSE POSITIVE: URI is checked by calling gettagno
```

Comment:
What does `gettagno` do?

Reply:
gettagno is a method in the class GeneralName of bouncycastle and the class has 9 final int fields with each of them having a tag number. gettagno() returns the tag number and here in the code it is checked if the tag is equal to that of a unifrom resource identifier (URI) field?